### DISCUSSION TITLE
Is there no model for clustering?

### DISCUSSION THREAD

**Discussion by DongHyukHwang** [timeseriesAI/tsai#442] [General]
Do you provide a clustering model rather than a classification model? 

(Written with Google Translate.)

**Comment by oguiza** [comment]
Hi @DongHyukHwang , 
No, there’s no specific clustering method in tsai at the moment.
There is however a method called ([MVP](https://github.com/timeseriesAI/tsai/blob/main/tutorial_nbs/08_Self_Supervised_TSBERT.ipynb)) that can be used to pretrain a model and generate learned representations (embeddings). Once you have t.hose embeddings you could use any dimensionality reduction technique (PCA, t-SNE, UMAP, …) to create the clusters.

**Comment by oguiza** [comment]
I'll move this question into discussions.
